Source/Resource/Plugin/Language/fr.csv

```diff
@@ -1230,133 +1230,133 @@ Subscribe_Failed;Achat annulé ou échoué
 Subscribe_Manage;Gérer l'abonnement
 Subscribe_Renew;Renouveler l'abonnement ?
 Subscribe_Renew_Price;Renouveler l'abonnement (%PRICE% pour un an)
-Subscribe_Unaviable;Impossible de récupérer les details de l'abonnement.
+Subscribe_Unaviable;Impossible de récupérer les détails de l'abonnement.
 Subscribe_Waiting;Abonnement en attente d'autorisation parentale
 Subscription_Active;Abonnement actif jusqu'au %DATE%
 Subscription_Android;Période d'essai gratuite de 3 jours.
 Subscription_Ended;Votre abonnement vient de se terminer.
 Subscription_Expired;Abonnement expiré depuis le %DATE%
-Subscription_Lifetime;Abonnement à vie detecté
+Subscription_Lifetime;Abonnement à vie détecté
 Subscription_Thanks;Abonnement détecté. Merci pour votre soutien !
-Substream;
+Substream;Sous-flux
 SubTrack;Sous-piste
-SubtstreamIdChannel;
+SubtstreamIdChannel;Identité de début de sous-flux et index de canal
 Summary;Résumé
 Supported formats;Formats supportés
-Supported?;Supporté?
-SupportUs;Soutenez-nous. Obtenez le thème sombre et d'autres fonctionnalités à venir.
-SurroundAttenuationKnown;
-SwitchGroup;
+Supported?;Supporté ?
+SupportUs;Nous Soutenir, Et obtenir le thème sombre ainsi que d'autres fonctionnalités à venir.
+SurroundAttenuationKnown;Atténuation surround connue
+SwitchGroup;Changer de groupe
 Synopsis;Synopsis
-SystemId;Id
+SystemId;Id système
 Tagged_Application;Application de marquage
 Tagged_Date;Date de marquage
-TargetDeviceConfig;
+TargetDeviceConfig;Configuration de l'appareil cible
 Technician;Technicien
 TermsOfUse;Conditions d'utilisation
-TertiaryAudioLanguage;
+TertiaryAudioLanguage;Langue audio tertiaire
 Text;Texte
-Text - Custom;Personnalisé
+Text - Custom;Personnaliser les textes
 Text (HTML);Texte (HTML)
-Text stream(s);Piste(s) texte
-Text streams;Flux texte
-Text_Codec_List;Codecs Texte
+Text stream(s);Sous-titre(s)
+Text streams;Sous-titres
+Text_Codec_List;Codecs sous-titres
 Text_No;Pas de sous-titres
-Text1;Première piste Texte
-Text2;Deuxième piste Texte
-Text3;Troisième piste Texte
-TextCount;Nombre de pistes texte
-TextlessElementsExist;
+Text1;Sous-titres 1
+Text2;Sous-titres 2
+Text3;Sous-titres 3
+TextCount;Nombre de pistes sous-titres
+TextlessElementsExist;Des éléments sans texte existent
 ThanksTo;Remerciements
 Thousands separator;Séparateur de milliers
-TimeCode;Time code
-TimeCode_FirstFrame;Time code de la première image
-TimeCode_LastFrame;
-TimeCode_MaxFrameNumber;
-TimeCode_MaxFrameNumber_Theory;
-TimeCode_Settings;Paramètres du time code
-TimeCode_Source;Source du time code
-TimeCode_Stripped;time code, calculé
-TimeStamp;Time stamp
+TimeCode;Index temporel

```

Comment:
"[Code temporel](https://fr.wikipedia.org/wiki/Timecode_(indicateur))".

Reply:
Tank's you for the Wiki link 
1275 and 1276 "Nombre d'images maximum du code temporel" and "Nombre d'images maximum théorique du code temporel"
is a personal view and I need you proposal